### PR TITLE
Notes on path volumes

### PR DESCRIPTION
**Pull request by AddoSolutions**
Notes regarding path volumes if you don't use a named volume

### CODE REVIEW

File: docs/docker-compose.md [hunk 1]
```diff
@@ -68,3 +68,11 @@ volumes:
     var:
     public:
 ```
+
+**You Must Use a Named Volume**
+
+In order for the "public" folder to get copied correctly, this must be a "named" volume, not a folder path. You will get a 404 if you do. If you MUST use a fodler path, this is what you must do:
```
Comment (jokay) [inline docs/docker-compose.md]: Typo: ... fo**dl**er ...

File: docs/docker-compose.md [hunk 2]
```diff
@@ -68,3 +68,11 @@ volumes:
...
+
+1. First, you need to change the volume map to `- ./my/folder:/opt/kimai/public2` (notice the "2" at the end)
+2. Now, `docker-compose up -d`, and then `docker-compose exec kimai bash` and inside that container `cd /opt/kimai`; cp -r public/* public2/; cp -r public/.htaccess public2`
```
Comment (jokay) [inline docs/docker-compose.md]: I would use a one-liner which can be copied, pasted & executed directly, e.g.

```sh
cd /opt/kimai && cp -r public/* public2/ && cp -r public/.htaccess public2
```

File: docs/docker-compose.md [hunk 3]
```diff
@@ -68,3 +68,11 @@ volumes:
     var:
     public:
 ```
+
+**You Must Use a Named Volume**
```
Comment (jokay) [inline docs/docker-compose.md]: Typos: ... You **m**ust **u**se a **n**amed **v**olume ...

**Comment by AddoSolutions** [comment]
It's hard to believe english IS my first language :)

**Comment by jokay** [comment]
> It's hard to believe english IS my first language

Well, not for me :wink:

**Comment by x-jokay** [comment]
LGTM 👍🏼

**Review by jokay — DISMISSED** [review]
LGTM 👍🏼

File: docs/docker-compose.md [hunk 4]
```diff
@@ -68,3 +68,11 @@ volumes:
     var:
     public:
 ```
+
+**You must use a named volume**
+
+In order for the "public" folder to get copied correctly, this must be a "named" volume, not a folder path. You will get a 404 if you do. If you MUST use a folder path, this is what you must do:
+
+1. First, you need to change the volume map to `- ./my/folder:/opt/kimai/public2` (notice the "2" at the end)
+2. Now, `docker-compose up -d`, and then `docker-compose exec kimai bash` and inside that container `cd /opt/kimai && cp -r public/* public2/ && cp -r public/.htaccess public2`
```
Comment (kevinpapst) [inline docs/docker-compose.md]: to be consistent I would add a trailing slash => `public2/`

File: docs/docker-compose.md [hunk 5]
```diff
@@ -68,3 +68,11 @@ volumes:
     var:
     public:
 ```
+
+**You must use a named volume**
+
+In order for the "public" folder to get copied correctly, this must be a "named" volume, not a folder path. You will get a 404 if you do. If you MUST use a folder path, this is what you must do:
```
Comment (kevinpapst) [inline docs/docker-compose.md]: > You will get a 404 if you do

Shouldn't that be better "You will get a 404 if you don't use a named volume"?

**Comment by AddoSolutions** [comment]
All those recommendations are wrapped up, LMK if you need anything else

**Review by tobybatch — APPROVED** [review]

**Pull request merged**